vocabularies/Common.xml

```diff
@@ -795,8 +795,15 @@ The minimum structure is
         </Property>
       </ComplexType>
       <ComplexType Name="ValueListParameterIn" BaseType="Common.ValueListParameter">
+        <Property Name="Operator" Type="UI.SelectionRangeOptionType" DefaultValue="0" Nullable="false">
+          <Annotation Term="Core.Description" String="Operator applied to the property. Defaults to `EQ`." />
+        </Property>
         <Property Name="LocalDataProperty" Type="Edm.PropertyPath" Nullable="false">
-          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list with `eq` comparison" />
+          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list by applying the operator (Single value or lower interval boundary)" />
+          <Annotation Term="Core.LongDescription" String="In case the property path contains a collection-based navigation or structural property, the filter is a set of `eq` comparisons connected by `or` operators" />
+        </Property>
+        <Property Name="LocalDataPropertyHigh" Type="Edm.PropertyPath" Nullable="false">
+          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list by applying the operator (Upper interval boundary)" />
           <Annotation Term="Core.LongDescription" String="In case the property path contains a collection-based navigation or structural property, the filter is a set of `eq` comparisons connected by `or` operators" />
```

Comment:
A collection of values for the upper boundary makes no sense.

Reply:
maybe we have here a misunderstanding ? this parameter only needs to be set if a developer wants to default the value to between, although I also don't have a meaningful use case for that, but when using the FE UI you are technically able to configure such scenario, therefore the vocabulary standard should not prohibit us to do so
![image](https://github.com/user-attachments/assets/3140f4eb-0815-4cc6-be73-0ed141ed42f8)

Reply:
Removed the BT use case and the upper boundary with 2e24e35